src/apiCalls.js

```diff
@@ -0,0 +1,24 @@
+const getClosetData = async () => {
```

Comment:
it would be useful to create a reusable error handling function that way we can easily set errors later
```
const handleError = (response) => {
  if (response.ok) {
    return response.json()
  } else {
    throw new Error(`Error ${response.status} -- Please try again`)
  }
}

```

Reply:
I added this function and called it in the getData- i havent tested getClosetData yet though- i wanted to continue working on the closet page FE to see if i can fetch the data and populate the app. lmk if this works!